Assets/MixedRealityToolkit/Definitions/SpatialAwareness/MixedRealitySpatialAwarenessMeshObserverProfile.cs

```diff
@@ -0,0 +1,136 @@
+        /// </summary>
+        public AutoStartBehavior StartupBehavior => startupBehavior;
+
+        [SerializeField]
+        [Tooltip("Should the spatial observer remain in a fixed location?")]
+        private bool isStationaryObserver = false;
+
+        /// <summary>
+        /// Indicates whether or not the spatial observer is to remain in a fixed location.
+        /// </summary>
+        public bool IsStationaryObserver => isStationaryObserver;
+
+        [SerializeField]
+        [Tooltip("The dimensions of the spatial observer volume, in meters.")]
+        private Vector3 observationExtents = Vector3.one * 3;
+
+        /// <summary>
+        /// The size of the volume, in meters per axis, from which individual observations will be made.
+        /// </summary>
+        public Vector3 ObservationExtents => observationExtents;
+
+        [SerializeField]
+        [Tooltip("The shape of observation volume")]
+        private VolumeType observerVolumeType = VolumeType.AxisAlignedCube;
+
+        /// <summary>
+        /// The shape (ex: axis aligned cube) of the observation volume.
+        /// </summary>
+        public VolumeType ObserverVolumeType => observerVolumeType;
+
+
+        [SerializeField]
+        [Tooltip("How often, in seconds, should the spatial observer update?")]
+        private float updateInterval = 3.5f;
+
+        /// <summary>
+        /// The frequency, in seconds, at which the spatial observer updates.
+        /// </summary>
+        public float UpdateInterval => updateInterval;
+
+        #endregion IMixedRealitySpatialAwarenessObserver settings
+
+        #region IMixedRealitySpatialAwarenessMeshObserver settings
+
+        [PhysicsLayer]
+        [SerializeField]
+        [Tooltip("Physics layer on which to set observed meshes.")]
+        private int meshPhysicsLayer = 31;
+
+        /// <summary>
+        /// The Unity Physics Layer on which to set observed meshes.
+        /// </summary>
+        public int MeshPhysicsLayer => meshPhysicsLayer;
+
+        [SerializeField]
+        [Tooltip("Level of detail used when creating the mesh")]
+        private SpatialAwarenessMeshLevelOfDetail levelOfDetail = SpatialAwarenessMeshLevelOfDetail.Coarse;
+
+        /// <summary>
+        /// The level of detail used when creating the mesh.
+        /// </summary>
+        public SpatialAwarenessMeshLevelOfDetail LevelOfDetail => levelOfDetail;
+
+        [SerializeField]
+        [Tooltip("Level of detail, in triangles per cubic meter.\nIgnored unless LevelOfDetail is set to Custom.")]
+        private int trianglesPerCubicMeter = 0;
+
+        /// <summary>
+        /// The level of detail, in triangles per cubic meter, for the returned spatial mesh.
+        /// </summary>
+        /// <remarks>This value is ignored, unless <see cref="LevelOfDetail"/> is set to Custom.</remarks>
+        public int TrianglesPerCubicMeter => trianglesPerCubicMeter;
```

Comment:
do we want a setter here or is this something that can only be set in an inspector/profile?